v2/test/sample_test.py

```diff
@@ -0,0 +1,82 @@
+# %%
+HOST = 'https://71a74112589c16e8-dot-asia-east1.pipelines.googleusercontent.com'
+CONTEXT = 'gs://gongyuan-pipeline-test/sample-test/src/context.tar.gz'
+SDK = 'gs://gongyuan-pipeline-test/sample-test/src/kfp-sdk.tar.gz'
+
+# %%
+import kfp
+import kfp.components as comp
+import json
+
+
+download_gcs_tgz = kfp.components.load_component_from_file('components/download_gcs_tgz.yaml')
+run_sample = kfp.components.load_component_from_file('components/run_sample.yaml')
+kaniko = kfp.components.load_component_from_file('components/kaniko.yaml')
+
+@kfp.dsl.pipeline(name='v2 sample test')
+def v2_sample_test(
+    context: 'URI' = CONTEXT,
+    launcher_destination: 'URI' = 'gcr.io/gongyuan-pipeline-test/kfp-launcher',
+    gcs_root: 'URI' = 'gs://gongyuan-test/v2',
+    samples_destination: 'URI' = 'gcr.io/gongyuan-pipeline-test/v2-sample-test',
+    kfp_host: 'URI' = 'http://ml-pipeline:8888',
+    samples_config: 'JSONArray' = json.dumps([
```

Comment:
JFYI: For KFP it's OK to just write it like this:
`samples_config: list = [...],`
The `list` type is converted to `JsonArray` and the `json.dumps` serialization is used.

Reply:
nice, I'll do it instead